js/src/get-started-page/index.js

```diff
@@ -5,12 +5,14 @@ import './index.scss';
 import UnsupportedNotices from './unsupported-notices';
 import Faqs from './faqs';
 import GetStartedCard from './get-started-card';
+import GetStartedWithVideoCard from './get-started-with-video-card';
 import FeaturesCard from './features-card';
 
 const GetStartedPage = () => {
 	return (
 		<div className="woocommerce-marketing-google-get-started-page">
 			<UnsupportedNotices />
+			<GetStartedWithVideoCard />
 			<GetStartedCard />
```

Comment:
❔ or 💅 
Would the GetStartedCard components be removed in another PR, or we could remove them along with this PR?

Reply:
Thanks for the reminder, there is a new card get `GetStartedCard` as well like in the design: eQ9O4m2flzrcAiDMXkOW0m-fi-7350%3A121253, so I will update it in another PR.